shell/app/modules/project/common/components/issue/edit-issue-drawer.tsx

```diff
@@ -1112,11 +1112,18 @@ export const EditIssueDrawer = (props: IProps) => {
   }
 
   if (!isEditMode) {
-    footer = [
+    footer = (isChanged: boolean, confirmCloseTip: string) => [
       <div key="holder" />,
       <Spin key="submit" spinning={updateIssueLoading}>
         <div>
-          <Button onClick={() => onClose()}>{i18n.t('cancel')}</Button>
+          {isChanged && confirmCloseTip ? (
+            <Popconfirm title={confirmCloseTip} placement="bottomRight" onConfirm={onClose}>
```

Comment:
is there any display issue if use  placement = `bottomRight` in footer
or should keep should behavior as esc key down to show a modal?